pack/rpm.mk

```diff
@@ -18,7 +18,12 @@ else
 RPMNAME := $(shell rpmspec -P $(RPMSPECIN) | sed -n -e 's/Name:\([\ \t]*\)\(.*\)/\2/p')
 endif
 
+ifneq ($(shell rpm -E "%{suse_version}"), %{suse_version})
+IS_SUSE := true
+RPMDIST := $(shell rpm -E ".sle%{sle_version}")
+else
 RPMDIST := $(shell rpm -E "%{dist}")
+endif
```

Comment:
I don't find any instruction that would suggest a right `Release` field value. Are you know one?

BTW, it affect a file name, right? I see that packages for leap 15.2 are names with `lp152` suffix [here](https://opensuse.pkgs.org/15.2/database-x86_64/). Is it good source of information? Some packages have suffix like `lp152.3.5`, don't know why.

I don't mind any way in fact, because I know nothing about SuSE, but it would be glad to base our way on some more or less widely adopted.

Reply:
I see, you added the comment:

```
# to check if SuSE OS is running than need to check the originaly
# used macro for SuSE versions, otherwise it will return the same
# https://en.opensuse.org/openSUSE:Packaging_for_Leap#RPM_Distro_Version_Macros
```

But it seems, we don't understand each other. First, I'm unable to parse 'to check <...> than need to check' phrase. Second, what 'originally' means in 'the originally used macro for SuSE versions'? What you want to say here? 'We use a macro that is expanded only on SuSE to define a custom `Release` field on this distro', right? It is obvious enough from the code, I think, and it is not what I asked above.

Please, read again my questions and ask me offline if they should be clarified.

I just want to understand why things are made this way and not another one. I see some inconsistency with naming of other packages and I want to know whether it appears by intention or by a mistake. This is why I asked you for your sources of information: we'll look at them together and decide how things should be implemented.

Reply:
Right, your release version naming is better and correct, check the following:
https://www.reddit.com/r/openSUSE/comments/gqw7sg/understanding_the_leap_package_versioning_scheme/
```
The release version prefix, e.g. lp152 for Leap 15.2, comes from the project configuration (https://build.opensuse.org/projects/openSUSE:Leap:15.2/prjconf). It is set by specifying the release version spec: spec:lp152.<CI_CNT>.<B_CNT>.